api/src/routes/pipelines.rs

```diff
@@ -292,6 +329,29 @@ pub struct GetPipelineReplicationStatusResponse {
     pub table_statuses: Vec<TableReplicationStatus>,
 }
 
+#[derive(Debug, Copy, Clone, Serialize, Deserialize, ToSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum RollbackType {
```

Comment:
For now, we agreed that the user can choose to reset to fully rollback to the initial state or just one state.